api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/OperationData.java

```diff
@@ -0,0 +1,33 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi;
+
+import java.util.Map;
+
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+
+public class OperationData {
+
+	private Map<String, String[]> params;
+	private ServletContext context;
+
+	public OperationData(HttpServletRequest request) {
+		params = request.getParameterMap();
```

Comment:
Pending the request VERB, will we not want to include doing something with the request body? And potentially headers. Or maybe just put the request on the OperationData, but then what purpose does the OperationData have yet. I assume the OperationData will hold additional "params" as it progresses through the action?